src/aatoolbox/datasources/usgs/usgs_ndvi.py

```diff
@@ -0,0 +1,924 @@
+"""Class to download and process USGS FEWS NET NDVI data.
+
+Data is downloaded from the `USGS FEWS NET data portal
+<https://earlywarning.usgs.gov/fews>`_. Data is
+generated from eMODIS AQUA, with full methodological
+details available from the various data pages
+on the data portal. The available areas of coverage are:
+
+- `North Africa<https://earlywarning.usgs.gov/fews/product/449>`_
+- `East Africa<https://earlywarning.usgs.gov/fews/product/448>`_
+- `Southern Africa<https://earlywarning.usgs.gov/fews/product/450>`_
+- `West Africa<https://earlywarning.usgs.gov/fews/product/451>`_
+- `Central Asia<https://earlywarning.usgs.gov/fews/product/493>`_
+- `Yemen<https://earlywarning.usgs.gov/fews/product/502>`_
+- `Central America<https://earlywarning.usgs.gov/fews/product/445>`_
+- `Hispaniola<https://earlywarning.usgs.gov/fews/product/446>`_
+
+Data is made available on the backend USGS file explorer. For example,
+dekadal temporally smooth NDVI data for West Africa is available at
+`this link
+<https://edcintl.cr.usgs.gov/downloads/sciweb1/shared/fews/web/africa/west/dekadal/emodis/ndvi_c6/temporallysmoothedndvi/downloads/monthly/>`_
+
+The products include temporally smoothed NDVI, median anomaly,
+difference from the previous year, and median anomaly
+presented as a percentile.
+
+Data by USGS is published quickly after the dekad.
+After about 1 month this data is updated with temporal smoothing
```

Comment:
Yes, thought about that last week and just didn't have brainspace to address yet. I think will need system to set clobber separately for recent, so that by default it re-downloads files in the last 1 month each time. It's not indicated in the file in any way, sadly, as they are just `.tif` files with a georeferencing `.tfw` file. Do you have any potential ideas on this? I'm still trying to think through.

If we just update the past 1 month, I think that's a safe bet. However, maybe I come back after a year and want to update. Could we myabe pass in `clobber_since` as an argument, that defaults to `None`, but when passed only clobbers files after that date?

Reply:
Pff horrible, this should be USGS responsibility. 
Buttt would just checking if the last modified date of the url is later than the download date? 

I did a bit of googling and the accepted answer [here](https://stackoverflow.com/questions/29314287/python-requests-download-only-if-newer) seems to be working. Where the returned `url_time` is the same as that indicated under Last Modified [on their website](https://edcintl.cr.usgs.gov/downloads/sciweb1/shared/fews/web/africa/east/dekadal/emodis/ndvi_c6/percentofmedian/downloads/dekadal/)

The date that is returned as `file_time` is the date the file was downloaded and thus not the one it was modified, but if we just check that that one is larger than `url_time` it shouldn't really matter. 

The computation of the times doesn't seem to take super long so hopefully wouldn't slow it down too much. 

I dont know if it is a solid approach and we should at least test it with a file that will get updated soon. But what do you think?

Reply:
That sounds like a really great approach, and makes loads of sense. I will look into that link you sent, thanks, that's brilliant!

Reply:
Really love the idea @Tinkaa!

Reply:
Done and implemented! Had to think through how to also adjust for this in the `process()` method. Let me know what y'all think.